src/cert.rs

```diff
@@ -37,6 +37,78 @@ pub struct Cert<'a> {
     pub subject_alt_name: Option<untrusted::Input<'a>>,
 }
 
+pub fn parse_root_cert_v1<'a>(cert_der: untrusted::Input<'a>)
```

Comment:
I would rather keep this in `trust_anchor_util`. Also, I think that this should just return a `TrustAnchor` directly, instead of constructing a `Cert` first. I think that would simplify it quite a bit. What do you think?